src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,48 @@
+.watch {
+  background: url("../images/clock.jpg") center no-repeat;
+}
+
+.clock {
+  height: 100vh;
+  @extend %place-content-center;
+  transform: rotate(180deg);
+
+  &__watch {
+    position: absolute;
+    top: 50%;
+
+    &--min {
+      height: $min-hand-height;
+      width: $hand-width;
+      background-color: $min-color;
+      animation: rotate $minute infinite;
+      transform-origin: center 0;
+      z-index: 1;
```

Comment:
Default value of `z-index` is 0. You shouldn't use z-index if the element is below all.